bigtop_toolchain/bin/puppetize.sh

```diff
@@ -20,51 +20,74 @@ if [ -f /etc/os-release ]; then
     . /etc/os-release
 fi
 
+RPM_DOWNLOAD_URL="https://yum.puppet.com/puppet7"
+APT_DOWNLOAD_URL="https://apt.puppet.com/puppet7"
+NIGHTLY=""
+
+if [ "${ARCH}" = "ppc64le" ];then
+RPM_DOWNLOAD_URL="https://nightlies.puppetlabs.com/yum/puppet7-nightly"
+APT_DOWNLOAD_URL="https://nightlies.puppetlabs.com/apt/puppet7-nightly"
+NIGHTLY="-nightly"
```

Comment:
does it[1][2] mean puppet7 is ready for ppc64le now?

[1] https://apt.puppet.com/dists/focal/puppet7/index.html
[2] https://yum.puppet.com/puppet7/el/8/index.html